wallet-extension/src/features/blockchain/balance.ts

```diff
@@ -0,0 +1,20 @@
+import type { WalletV1 } from "@nilfoundation/niljs";
+
+export async function fetchBalance(wallet: WalletV1): Promise<bigint> {
+  try {
+    return await wallet.getBalance();
```

Comment:
smart-account

Reply:
You caught me 😂 I opened pr before refactor i probably should go over it and see if i need to rename somewhere else

Reply:
Done: https://github.com/NilFoundation/nil/compare/5245c137498a24c3b68692459869e19562559678..69753d2ece4523198afee5843bcb27d69f7c4ea9